src/App.js

```diff
@@ -2,14 +2,52 @@ import React from 'react';
 import Header from './Header.js';
 import Main from './Main'
 import Footer from './Footer.js';
+import SelectedBeast from './SelectedBeast.js'
 import beasts from './data.json';
+import { Modal } from 'bootstrap';
```

Comment:
Good work on the lab? Do we need this named import? I think we are using Modal in `selectedBeast` :)